### DISCUSSION TITLE
Change Option background color.

### DISCUSSION THREAD

**Discussion by itxchandan** [JedWatson/react-select#5053] [Q&A]
I want to change the background color from the **options**. Please assist.

![image](https://user-images.githubusercontent.com/17069784/153185101-3785f76e-7c84-45fc-8234-4a9b04904f04.png)

**Comment by manjushsh** [comment]
@itxchandan Please refer [this](https://react-select.com/components)

Pass custom CSS to your select
```
const reactSelectStyles = {
  option: (provided, state) => ({
    ...provided,
    ":hover": {
      backgroundColor: "red",
      color: "white"
    }
  }),
};
```
and on select
```
 <Select
   value={option}
   onChange={updateOption}
   options={options}
   styles={reactSelectStyles}
 />
```
